test/unit/next-babel-loader-prod.test.ts

```diff
@@ -30,6 +30,7 @@ const babel = async (code: string, queryOpts = {} as any) => {
           : path.resolve(dir, 'pages'),
       cache: false,
       hasReactRefresh: false,
+      pageExtensions: ['tsx', 'ts', 'jsx', 'js'],
```

Comment:
```suggestion
    nextConfig: { pageExtensions: [] }
```

We could keep this test suite without extensions to also test that case